app/src/main/java/com/flowfoundation/wallet/page/send/transaction/subpage/amount/presenter/SendAmountPresenter.kt

```diff
@@ -78,10 +79,27 @@ class SendAmountPresenter(
     }
 
     override fun bind(model: SendAmountModel) {
-        model.balance?.let { updateBalance(it) }
+        model.balance?.let {
+            updateBalance(it)
+            if (!initialAmountSet) {
+                setInitialAmount()
+                initialAmountSet = true
+            }
+        }
         model.onCoinSwap?.let { updateCoinState() }
     }
 
+    private fun setInitialAmount() {
+        viewModel.getInitialAmount()?.let { amount ->
+            if (amount.isNotBlank()) {
+                with(binding) {
+                    transferAmountInput.setText(amount)
+                    transferAmountInput.setSelection(transferAmountInput.text.length)
```

Comment:
For hex value, we have do the conversion using 18 decimals ?
For example: 
EVM using `0xDE0B6B3A7640000` means `1000000000000000000`.
But we need show `1.0` to the user